app/models/template.rb

```diff
@@ -146,6 +150,10 @@ def import_oses(options)
     end
   end
 
+  def import_taxonomy_override(taxonomy_params)
+    taxonomy_params.map  { |key, value| self.public_send("#{key}=", value) }
```

Comment:
this does not seem very readable, I'd prefer to be more explicit and go over all enabled taxonomies https://github.com/theforeman/foreman/blob/develop/app/models/taxonomy.rb#L91